userguide/jails.rst

```diff
@@ -759,17 +727,18 @@ entries available for a jail.
 Accessing a Jail Using SSH
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Accessing a jail is not limited to using Jail Add, the Jail Wizard, or
-using one of the Shells that are built in to the UI. Enable the ssh
-daemon `sshd(8)
-<https://www.freebsd.org/cgi/man.cgi?query=sshd&apropos=0&sektion=8>`__
-in a jail, create a user account to log in with, and ssh access to
-the jail via any computer that has network access to the %brand% server
-is now possible.
 
-Log in to a shell to get :guilabel:`cli` access to the jail to enable SSH
-on. To use the Jails Shell click |ui-options| --> :guilabel:`Shell` on
-the desired jail. A root shell will open as shown in this example:
+The ssh daemon
+`sshd(8) <https://www.freebsd.org/cgi/man.cgi?query=sshd>`__
+must be enabled to access a jail from another system using SSH.
+
+Access the shell of the jail in the %brand% UI to enable SSH. Log into
+the UI. Click
+:menuselection:`Jails -->` |ui-options| :menuselection:`--> Shell`
+for the desired jail. The jail status must be :guilabel:`up` for
+:guilabel:`Shell` to be available. Start the jail by clicking
+|ui-options| and :guilabel:`Start`. The jail root shell is shown in this
+example:
```

Comment:
The ordering of sentences in this paragraph is weird.  It starts with telling the user to open a shell, then tells them the jail must be up, then tells them to start the jail.  This needs to be simplified.
1. The jail must be up ("running"?  whatever the exact thing shown in the UI says).
2. If the jail is not up, start it by doing this.
3. Click this to start a shell on the jail.